bbb/test/dbutils.py

```diff
@@ -55,3 +55,17 @@ def makeSchedulerDb(db):
                 `results` SMALLINT
             );
 """))
+
+
+def makeBBBDb(db):
+    db.execute(sa.text("""
+CREATE TABLE tasks (
+                `buildrequestId` INTEGER,
+                `taskId` VARCHAR(32) NOT NULL,
+                `runId` INTEGER NOT NULL,
+                `createdDate` INTEGER,
+                `processedDate` INTEGER,
+                `takenUntil` INTEGER,
+                CONSTRAINT bbb_taskId_runId UNIQUE (taskId, runId)
+            );
+"""))
```

Comment:
I think we still need an index on this column for e.g. getTaskFromBuildRequest, updateRunId, updateTakenUntil and deleteBuildRequest.

Does the UNIQUE constraint imply indexes on taskId/runId as well?